twitter.html

```diff
@@ -134,44 +131,38 @@ <h1 class="border transparent">Home</h1>
           </div>
           <div class="border two_column_grid">
             <ul class="list list_items_in_row">
-
               <li class="border list sm_box">
                 <div class="icon material-icons-outlined blue_icon hover_blue">
-                    image
+                  image
                 </div>
               </li>
-
               <li class="border list sm_box">
                 <div class="icon material-icons-outlined blue_icon hover_blue">
                   gif_box
                 </div>
               </li>
-
               <li class="border list sm_box">
                 <div class="icon material-icons-outlined blue_icon hover_blue">
                   poll
                 </div>
               </li>
-
               <li class="border list sm_box">
                 <div class="icon material-icons-outlined blue_icon hover_blue">
                   sentiment_satisfied_alt
                 </div>
               </li>
```

Comment:
This part took me a while to figure out.
Problem: The icons got padding when hovering over, therefore increasing width and pushing the whole row right side.
Solution: Wrapped each icon into a div, the .sm_box has a fixed width now and the .hover.blue icon always stays in a fixed position centred in the container.